client/pages/social.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="w-full h-full overflow-y-auto px-2 py-18 md:p-8">
+    <div class="flex flex-col md:flex-row">
+      <div class="w-200 my-6 mx-auto">
+        <div class="grid grid-cols-2 content-between">
+          <h1 class="text-2xl mb-4 font-book">Social</h1>
+          <div class="flex py-1 justify-end">
+            <ui-tooltip text="Share your latest listening activity with other users in the social tab">
+              <p class="pl-4">
+                Share Listening Activity
+                <span class="material-icons icon-text text-sm">info_outlined</span>
+              </p>
+            </ui-tooltip>
+            <ui-toggle-switch v-model="isSharingActivity" @input="changeSharingSetting" />
+          </div>
+        </div>
+        <p v-if="!users.length">No Other User Sessions</p>
+        <template v-for="(user, num) in users">
+          <div :key="user.username" class="w-full py-0.5">
+            <div class="flex items-center mb-1">
+              <nuxt-link :to="`/item/${user.latest.libraryItemId}`">
+                <ui-tooltip direction="left" :text="Math.round(user.latest.progress * 100) + '%'">
+                  <covers-book-cover class="absolute left-0 right-0" :library-item="user.libraryItem" :width="bookCoverWidth" :book-cover-aspect-ratio="bookCoverAspectRatio" />
+                </ui-tooltip>
+              </nuxt-link>
+              <p class="text-lg px-2 font-book text-white">{{ user.username }}&nbsp;</p>
+              <p class="text-xs text-white text-opacity-50">{{ "Last listened " + $dateDistanceFromNow(user.latest.lastUpdate) }}</p>
+              <div class="flex-grow" />
+              <div class="w-30 pl-4 text-right">
+                <p class="text-sm font-bold">{{ "Has listened for " + $elapsedPrettyExtended(user.minutesListened) }}</p>
```

Comment:
This would be the users all-time listening time and not just the listening time for the current session.
The UI was confusing to me because the username was next to the cover where I would expect the library item title to be.

Reply:
The cover is a link to the item's page and shows percentage complete on hover. Do you think all-time listening minutes or latest session would be best? It is currently all-time minutes